src/System.Diagnostics.Process/tests/RemotelyInvokable.cs

```diff
@@ -161,6 +161,17 @@ public static int Write144Lines()
             return SuccessExitCode;
         }
 
+        public static int WriteLinesSlowlyToOutputAndError()
+        {
+            for (int i = 0; i < 50; i++)
+            {
+                Console.WriteLine("This is line #" + i + ".");
+                Console.Error.WriteLine("This is error line #" + i + ".");
+                Thread.Sleep(100);
```

Comment:
Rather than add 5 seconds to the test duration (assuming the test waits on the remote invoke) would it be just as good to have the child process wait on an event indicating the Process has been closed, and then send one line to output and error and return?